src/microbit/test/test_image.py

```diff
@@ -0,0 +1,288 @@
+import pytest
+from ..model.image import Image
+
+from ..model import constants as CONSTANTS
+
+
+class TestImage(object):
+    def setup_method(self):
+        self.image = Image()
+        self.image_heart = Image(CONSTANTS.HEART)
+
+    @pytest.mark.parametrize("x, y, brightness", [(1, 1, 4), (2, 3, 6), (4, 4, 9)])
+    def test_get_pixel(self, x, y, brightness):
+        self.image._Image__LED[y][x] = brightness
+        assert brightness == self.image.get_pixel(x, y)
+
+    @pytest.mark.parametrize("x, y, brightness", [(1, 1, 4), (2, 3, 6), (4, 4, 9)])
+    def test_set_pixel(self, x, y, brightness):
+        self.image.set_pixel(x, y, brightness)
+        assert brightness == self.image._Image__LED[y][x]

```

Comment:
I would merge these two into one test and only test `set_pixel/get_pixel` instead of `__LED`.
In general we want to focus on testing behavior, rather than implementation. If we change the implementation we don't want to have tests failing.